CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 0.5.0
+
+### Features
+
+- Improved functionality for uploading links to images. This will allow the bot to send you your picture in response, and in the future, it will optimize the forwarding of messages with attachments to other chats.
```

Comment:
@Oleksandr1414  could you please add a description and some notes on what technology was used for attachments of video items